pkg/generictypes/generictypes.go

```diff
@@ -90,6 +92,8 @@ func GenericTypeFromString(arg string) int {
 		return GenericKiocb
 	case "iov_iter":
 		return GenericIovIter
+	case "load_info":
```

Comment:
I don't see 'lad_info' type being used later in the example tracing policy or mentioned in the doc, do we really need it? maybe 'module' type might be enough?

Reply:
> I don't see 'load_info' type being used later in the example tracing policy or mentioned in the doc, do we really need it? maybe 'module' type might be enough?

Yes I may use it in follow up restrict module loading killing or when inspecting signature. The load_module() https://elixir.bootlin.com/linux/latest/source/kernel/module/main.c#L2820  first parses the module into load_info then construct the rest into load_info->module, and the structures have fields that point to same location  load_info->name and module->name also load_info->sig_ok and module->sig_ok but this last one is a config compile time... this logic and function changes a lot and to give ourselves margin I added both, so we can easily produce tracing policies without new code change or making a new release

Reply:
From the user space perspective they both return a same struct [kernelmodule](https://deploy-preview-1390--tetragon.netlify.app/docs/reference/grpc-api/#kernelmodule) which I think covers what we want from the loading path

Reply:
Forget to mention during development: I added one and another and vise versa and was short depending on kernel versions, did test only some kernel versions, but on one ubuntu kernel I think that path was patched since I missed the load_info->sig_ok for no obvious reasons...  so just concluded lets add both and we can adapt ;-)